src/System.Globalization/tests/CompareInfo/CompareInfoTests.Compare.cs

```diff
@@ -256,20 +253,99 @@ public static IEnumerable<object[]> Compare_Random_TestData()
                 yield return new object[] { s_currentCompare, string1, string2, CompareOptions.Ordinal, PredictCompareOrdinalResult(string1, string2) };
             }
         }
-        
+
         [Theory]
-        [MemberData(nameof(Compare_TestData))]
+        [MemberData(nameof(Compare_Basic_TestData))]
+        [MemberData(nameof(Compare_Random_TestData))]
```

Comment:
FYI this was the problematic addition (the test data already existed but wasn't enabled so I enabled it). Sorry